<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7LGPRPJ" failures="1" tests="1" name="extent Report Tests" time="8.604" errors="0" timestamp="2024-02-05T20:28:26 EET">
  <testcase classname="smokeTests.loginPage" name="login" time="8.194">
    <failure type="java.lang.AssertionError" message="expected [https://magento.softwaretestingboard.com/] but found [https://magento.softwaretestingboard.com/customer/account/login/referer/aHR0cHM6Ly9tYWdlbnRvLnNvZnR3YXJldGVzdGluZ2JvYXJkLmNvbS8%2C/]">
      <![CDATA[java.lang.AssertionError: expected [https://magento.softwaretestingboard.com/] but found [https://magento.softwaretestingboard.com/customer/account/login/referer/aHR0cHM6Ly9tYWdlbnRvLnNvZnR3YXJldGVzdGluZ2JvYXJkLmNvbS8%2C/]
at pageObjects.Assertions.checkCurrentURLIsContain(Assertions.java:18)
at smokeTests.loginPage.login(loginPage.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- extent Report Tests -->
